components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -194,9 +194,32 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
                             }}
                         />
                     ) : (
-                        <FormattedMarkdownMessage
+                        <FormattedMessage
                             id='admin.feature_discovery.trial-request.accept-terms'
-                            defaultMessage='By clicking **Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails.'
+                            defaultMessage='By clicking <highlight>Start trial</highlight>, I agree to the <linkEvaluation>Mattermost Software Evaluation Agreement</linkEvaluation>, <linkPrivacy>Privacy Policy</linkPrivacy>.'
```

Comment:
Was `, and receiving product emails` intended to be removed from the message?

Reply:
Oops... copy paste error. Fixed it!